app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt

```diff
@@ -1,16 +1,337 @@
+    ) {
+        Checkbox(
+            checked = isChecked.value,
+            onCheckedChange = { isChecked.value = it }
+        )
+        Text(text = stringResource(R.string.admin_label))
+    }
+}
+
+
+@Composable
+fun ProfileIcon() {
+    val painter = rememberAsyncImagePainter(R.drawable.ic_baseline_person_24)
+
+    Column(
+        modifier = Modifier
+            .padding(8.dp)
+            .fillMaxWidth(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(
+            painter = painter,
+            contentDescription = null,
+            modifier = Modifier
+                .wrapContentSize()
+                .size(150.dp),
+            contentScale = ContentScale.Crop
+        )
+    }
+}
+
+@Composable
+fun ButtonRow() {
+    val notification = rememberSaveable() { mutableStateOf("") }
+    if (notification.value.isNotEmpty()) {
+        Toast.makeText(LocalContext.current, notification.value, Toast.LENGTH_SHORT).show()
+        notification.value = ""
+    }
+
+    Column(
+        modifier = Modifier
+            .verticalScroll(rememberScrollState())
+            .padding(8.dp)
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Card(
+                shape = CircleShape,
+                backgroundColor = CrayolaBrown,
+                modifier = Modifier
+                    .padding(8.dp)
+                    .size(50.dp)
+            ) {
+                IconButton(
+                    onClick = { notification.value = "Profile updated" },
+                ) {
+                    Icon(
+                        imageVector = Icons.Filled.SaveAlt,
+                        null,
+                        tint = Color.White,
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
+            }
+
+            Card(
+                shape = CircleShape,
+                backgroundColor = CrayolaBrown,
+                modifier = Modifier
+                    .padding(8.dp)
+                    .size(50.dp)
+            ) {
+                IconButton(
+                    onClick = {
+                        notification.value = "Profile deleted"
+                    },
+                ) {
+                    Icon(
+                        imageVector = Icons.Filled.Delete,
+                        null,
+                        tint = Color.White,
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
+            }
+            Card(
+                shape = CircleShape,
+                backgroundColor = CrayolaBrown,
+                modifier = Modifier
+                    .padding(8.dp)
+                    .size(50.dp)
+            ) {
+                IconButton(
+                    onClick = { notification.value = "Cancelled" },
```

Comment:
Use the american version `canceled`. This is a general rule of thumb that the american version is preferred. The same is the case for words like `color` (american) vs `colour` (british).

Reply:
In addition to that I think the cancel-button should be removed.
We use cancel inside the app to cancel actions like pop-ups or bottomDrawers.
This button would break the consistency. Since the profileView is a View we would also need a cancel-button in every other View. Since we have a NavBar the user could navigate to other views via the NavBar or with the back-action.

Reply:
Should be done with 797db1b89abf86077190bea3da15f309b0ea79a9